src/objects/Player.gd

```diff
@@ -0,0 +1,22 @@
+extends KinematicBody
+
+
+const MOUSE_SENS = Vector2(0.0025, 0.0025)
+
+onready var camera := $"%Camera" as Camera
```

Comment:
Lmao github is parsing the gdscript wrong, it's not ending the string with this quote